docs/pages/dsDisplay/ds-display_L3_Test-Procedure.md

```diff
@@ -0,0 +1,305 @@
+dsDisplay_test02_TestVerifyDisplayEdid.py --config /host/tests/configs/example_rack_config.yml --deviceConfig /host/tests/configs/deviceConfig.yml
+```
+
+- The test will download all required artifacts, copy them to the target directory, and start execution.
+
+- Display Device Verification:
+
+  - For each port, it will prompt the user to connect the specified display device and confirm successful connection.
+  - The test will retrieve the `EDID` information from the connected display device and verify it against the expected details, such as the monitor name, for each display.
+
+- Completion and Results:
+
+  After verifying the `EDID` information for all connected display devices, the test will conclude and display the final result (PASS/FAIL).
+
+### dsDisplay_test03_AspectRatioVerificationTest.py
+
+**Overview:**
+
+This test retrieves and verifies the aspect ratio of the display on each supported video port. For each specified aspect ratio, the test checks that the retrieved aspect ratio matches the expected value, ensuring that the display correctly adapts to the given configuration.
+
+**Platform Supported:**
+
+- Source
+
+**User Input Required:**
+
+Yes: User input is required to manually set the aspect ratio for the specified port, with the prompt question `Set the aspect ratio {aspectRatio} to Port: {port} and Press Enter:` (This will be automated later).
+
+**Acceptance Criteria:**
+
+Verify that the display’s aspect ratio is correctly retrieved for each configured setting (16x9, 4x3).
+
+**Expected Results:**
+
+The test will set various aspect ratios, retrieve the current aspect ratio from the display, and confirm that it matches the expected value. The test will pass if all aspect ratios are correctly identified; otherwise, it will fail.
+
+**Test Steps:**
+
+- Run the Python file `dsDisplay_test03_AspectRatioVerificationTest.py` with the appropriate configuration:
+
+```bash
+dsDisplay_test03_AspectRatioVerificationTest.py --config /host/tests/configs/example_rack_config.yml --deviceConfig /host/tests/configs/deviceConfig.yml
+```
+
+- The test will download the required artifacts, then copy them to the target directory.
+
+- Aspect Ratio Verification:
+    - For each video port, the user will be prompted to set the specified aspect ratio on the port and confirm by pressing Enter.
+    - The test will retrieve the aspect ratio from the connected display device.
+    - The retrieved aspect ratio will be compared to the expected value for each configuration.
+
+- Completion and Results:
+
+  If all expected aspect ratios are verified for each port, the test will conclude and display the final result (PASS/FAIL).
+
+### dsDisplay_test04_TestVerifyDisplayEdidBytes
+
+**Overview:**
+
+This test retrieves and verifies the `EDID` bytes from connected displays through each supported video port. The test compares the retrieved EDID bytes against expected values to ensure accurate identification and functionality of the display.
```

Comment:
Explain this for both source and sink devices and how it will consider the data.  We shall not connect the display units for a sink platform.  So be specific how the connection goes for a source and sink devices.